pkg/cfn/manager/api.go

```diff
@@ -262,6 +298,7 @@ func (c *StackCollection) doCreateChangeSetRequest(i *Stack, action string, desc
 		StackName:     i.StackName,
 		ChangeSetName: &changeSetName,
 		Description:   &description,
+		RoleARN: aws.String(c.cfnSvcRoleArn),
```

Comment:
I'd just use `&c.cfnSvcRoleArn` here, which matches the way it's done above.

Reply:
Agreed! But I ended up removing this line due to the redundancy 0bea25c